scripts/utils/render.js

```diff
@@ -100,7 +103,17 @@ const packageFormBuilder = () => {
 };
 
 const projectsFormBuilder = () => {
-  return;
+    const domString = `
+        <form>
+            <label for="board-name">Project board name</label><br>
```

Comment:
Change ids at 109, 111, and 112 to camelCase (ie "boardName")

Otherwise, it all looks good!